app/javascript/components/App.js

```diff
@@ -14,14 +14,37 @@ import {
 } from 'react-router-dom'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+
+  componentDidMount(){
+    this.readApartment()
+  }
+
+  readApartment = () => {
+    fetch("http://localhost:3000/apartments")
```

Comment:
Interestingly, we can just fetch to `/apartments` since you are not going across an origin like in Cat Tinder.

Reply:
thank you, that makes sense! I was wondering about that since it is the same app